segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/DebugBookKeeperLogWrapper.java

```diff
@@ -264,6 +267,19 @@ public void deleteDurableLogMetadata() throws DurableDataLogException {
         }
     }
 
+    public void deleteLedgersStartingWithId(long startId) throws DurableDataLogException {
+        LogMetadata metadata = this.log.loadMetadata();
+        List<LedgerMetadata> ledgers = metadata.getLedgers();
+        List<Long> ids = ledgers.stream()
+                .map(ledgerMetadata -> ledgerMetadata.getLedgerId())
+                .collect(Collectors.toList());
+        if (!ids.contains(startId)) {
+            throw new DurableDataLogException("No such ledger exist in log: " + this.log.getLogId() );
```

Comment:
Unnecessary space at the end of this line. Also, adding the ledger id in the exception message would be useful for understanding the problem if this exception is thrown.

Reply:
Updated.